ideascube/monitoring/models.py

```diff
@@ -127,6 +127,10 @@ class Specimen(models.Model):
 
     objects = SpecimenQuerySet.as_manager()
 
+    @property
+    def user_identifier(self):
```

Comment:
`user_identifier` is a pretty poor name for this, since `user` generally refers to the user model, the logged in user, etc...

I generally see things like `display_*` or `pretty_*` being commonly used in such cases.